### ISSUE TITLE
Plugin does not open files automatically.

### ISSUE THREAD

**Issue by nwehrle** [chrisgrieser/shimmering-obsidian#1]
Do I miss any obvious configuration steps with the URI-Plugin?
1) If I do the "odual" it states that the Vault has not been found.
2) If I do the OCR, the file will not be opened, even if a msg states this.

**Comment by nwehrle** [comment]
Maybe related, not sure, if I try to "odual", a message appears, vault not found.

**Comment by nwehrle** [comment]
Also the needed CLI utils have been installed. I use tesseract to ocr into the commandline and it works properly.

**Comment by chrisgrieser** [comment]
> the file will not be opened (URI Plugin ist installed)
> if I try to "odual", a message appears, vault not found.

This sounds like you might have entered an incorrect vault path in your workflow configuration. You must enter the **full absolute path** to you vault, which means something like `/User/pseudometa/MyVault` – mind the leading `/` and _leave out_ any `/` at the end (`~` for your home folder is fine). Do *not* escape any characters.

> The file is more or less empty (contains just the date)

This most likely means that there is some issue with tesseract. Have you entered appropriate language codes in the workflow configuration under `ocr_languages`? (Also there is a rare bug with macOS' screenshot utility when you take screenshots while having a hidden accessibility keyboard.)

If your issue still persists after checking all this, I would need further information to figure out your issue as the beta testers did reported the OCR-feature to be working fine. Would you mind sharing some screenshots or a screen recording? Alternatively/Additionally, a debugging log would also be very helpful.

(You can get a debugging log by opening the workflow in Alfred preferences and pressing `cmd + D`. A small window will open up which will log everything happening during the execution of the Workflow. Use the malfunctioning part of the workflow once more, copy the content of the log window.)

**Comment by nwehrle** [comment]
Sorry for the confusion. The OCR works now (OSX security asks to record the screen). The message was hidden in the background. The "odual" problem stays.
The configured path is a bit long and looks like this: "/Users/xxx/Library/Mobile Documents/iCloud~md~obsidian/Documents/MDVault"

**Comment by nwehrle** [comment]
Is the tilde a problem?
<img width="501" alt="Screenshot 2021-08-28 at 23 36 12" src="https://user-images.githubusercontent.com/832234/131231604-fa2dfaf1-ad46-495f-ac6a-85e73356b293.png">

**Comment by nwehrle** [comment]
odual ->
<img width="249" alt="Screenshot 2021-08-28 at 23 37 51" src="https://user-images.githubusercontent.com/832234/131231637-33006c64-fd51-4284-925b-05dcbce4ce82.png">

```
[23:37:38.412] Shimmering Obsidian[Script Filter] Queuing argument '(null)'
[23:37:38.414] Shimmering Obsidian[Script Filter] Queuing argument '(null)'
[23:37:40.524] Shimmering Obsidian[Keyword] Processing complete
[23:37:40.525] Shimmering Obsidian[Keyword] Passing output '' to Run Script
[23:37:40.758] Shimmering Obsidian[Run Script] Processing complete
[23:37:40.759] Shimmering Obsidian[Run Script] Passing output '' to Run Script
```

**Comment by chrisgrieser** [comment]
Got it, it was only a small issue with the URI Scheme used by `odual`. Problem is fixed with [version 1.2](https://github.com/chrisgrieser/shimmering-obsidian/releases/tag/1.2), which I just uploaded. :)

> Sorry for the confusion. The OCR works now (OSX security asks to record the screen). The message was hidden in the background. 

Ah, nevertheless good point. I added a brief notice in the documentation that you have to give Alfred permission to record the screen. Thanks.
